indico/web/templates/layout/base.html

```diff
@@ -5,6 +5,13 @@
                 <div class="title">
                     <div class="text">
                         <div class="title-with-actions">
+                            {% if self.back_button_url() %}
```

Comment:
`self.back_button_url()|trim` to make it immune to blocks containing just whitespace